tests/testthat/test-Rules-methods.R

```diff
@@ -173,3 +173,132 @@ test_that("StoppingMTDCV works correctly if CV is above threshold", {
   )
   expect_identical(result, expected) # CV is 23% > 20%.
 })
+
+
+# stopTrial-StoppingLowestDoseHSRBeta ----
+
+my_data <- h_get_data()
+
+# Model and samples are not needed to evaluate the rule, but needed for stopTrial
+# and therefore only generated once.
+my_model <- LogisticKadane(0.3, xmin = 0.001, xmax = 100)
+my_options <- McmcOptions(
+  burnin = 1, step = 1, samples = 1, rng_kind = "Mersenne-Twister", rng_seed = 94
+)
+my_samples <- mcmc(my_data, my_model, my_options)
```

Comment:
hm yeah I did not mention this above it seems, but in general this makes running individual tests harder. I would recommend
1) either use only one set of constant data sets defined on top of file. and then change/adapt inside each test
or
2) put this in a little helper function and then call this function within each of the tests below to recreate these objects, (e.g. return them in a list)